iceoryx_utils/include/iceoryx_utils/internal/concurrent/lockfree_queue/index_queue.hpp

```diff
@@ -177,6 +177,10 @@ class IndexQueue
     using Index = CyclicIndex<Capacity>;
 
     using Cell = std::atomic<Index>;
+    /// note: this member has to be initialized explicitly in the constructor since

```

Comment:
please use @note here for doxygen

Reply:
I removed the note since this is not part of the developer documentation. It is a hint for fellow developers to explicitly initialise this array with a loop when they implement a new constructor.